roles/ceph-container-common/tasks/pre_requisites/prerequisites.yml

```diff
@@ -93,7 +93,7 @@
 - name: ensure tmpfiles.d is present
   lineinfile:
     path: /usr/lib/tmpfiles.d/ceph-common.conf
```

Comment:
Your commit said `- move file to /etc/tmpfiles.d`, that doesn't seem to be the case

Reply:
thanks maggle